webhook_to_fedora_messaging/move.py

```diff
@@ -0,0 +1,145 @@
+    async for sess in get_session():
+        for item in read_repo():
+            name, desc, lang, team, active = item
+            try:
+                flag = 0
+                if team in pairdata:
+                    flag = 1
+                else:
+                    try:
+                        flag = 2
+                        qery = select(User).filter_by(name=team).options(selectinload("*"))
+                        rslt = await sess.execute(qery)
+                        user = rslt.scalar_one()
+                    except NoResultFound:
+                        print(f"[{datetime.now().strftime(format)}] Service '{team}/{name}' abandoned.")
+                        continue
+
+                # SERVICE IS TO BE CREATED ONLY IF FLAG IS 1 OR 2
+                # ORPHANS ARE TO BE ABANDONED
+                make_service = Service(
+                    name=f"{team}/{name}",
+                    uuid=uuid4().hex[0:8],
+                    type="github",
+                    desc=f"Written in {lang}. {desc}" if lang != "None" else f"{desc}",
+                    disabled=False if active == "t" else True
+                )
+                sess.add(make_service)
+                await sess.flush()
+                await sess.commit()
+
+                if flag == 1:
+                    if team not in done:
+                        for username in pairdata[team]:
+                            qery = select(User).filter_by(name=username).options(selectinload("*"))
+                            rslt = await sess.execute(qery)
+                            user = rslt.scalar_one()
+                            stmt = owners_table.insert().values(
+                                {"service_id": make_service.id, "user_id": user.id}
+                            )
+                            await sess.execute(stmt)
+                            await sess.commit()
+                            done.add(team)
+                elif flag == 2:
+                    qery = select(User).filter_by(name=team).options(selectinload("*"))
+                    rslt = await sess.execute(qery)
+                    user = rslt.scalar_one()
+                    stmt = owners_table.insert().values(
+                        {"service_id": make_service.id, "user_id": user.id}
+                    )
+                    await sess.execute(stmt)
+                    await sess.commit()
+
+                print(f"[{datetime.now().strftime(format)}] Service '{team}/{name}' created.")
+            except Exception as expt:
+                await sess.rollback()
+                print(f"[{datetime.now().strftime(format)}] Service '{team}/{name}' failed.")
+                print(f"An error occurred: {expt}")
+            finally:
+                await sess.close()
+
+
+if __name__ == "__main__":
+    set_config_file("/etc/webhook-to-fedora-messaging/webhook-to-fedora-messaging.cfg")
+    conf = get_config()
+    # run(import_user_to_database())
```

Comment:
Is this a debug leftover, or do we not want to run this function?

Reply:
Both of these need to run but to control the course - I was running one at a time.

I guess this can be uncommented so that both of these things run in succession.